german/schwimmen/schwimmen.md

```diff
@@ -4,13 +4,16 @@ Sie erreichen Sie mit letzter Kraft und erholen sich erst einmal am Strand. Doch
 
 Man sollte ...
 
+... etwas scharfes finden, vielleicht einen Stein, 
+und diese Fischerleine aus den Flügeln kriegen.
+[Flügel](fluegel_freischneiden/fluegel_freischneiden.md)
+
 ... was zu Essen suchen, denn nach dem Schwimmen hat man immer Hunger
 [Hunger](hunger/hunger.md)
 
 ... ein sicheres Lager aufbauen, denn wer weiß schon welche Gefahren auf einer Insel lauern
 [Gefahr](gefahr/gefahr.md)
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.